lib/BaseController.js

```diff
@@ -111,7 +114,7 @@ module.exports = class BaseController {
   }
 
   errorHandler(err) {
-    this._logger.error(err, this._data);
+    this._logger.error(`${this.selfLink}: `, err);
```

Comment:
```suggestion
    this._logger.error(err, this.selfLink);
```

Reply:
doesnt work because err can be an array of errors and bunyan doesnt interpret it as an error, and it prints backwards.